finality-aleph/src/sync/data.rs

```diff
@@ -27,17 +27,62 @@ impl<J: Justification> State<J> {
     }
 }
 
+/// Additional information about the branch connecting the top finalized block
+/// with a given one. All the variants are exhaustive and exclusive due to the
+/// properties of the `Forest` structure.
+#[derive(Clone, Debug, Encode, Decode, PartialEq, Eq)]
+pub enum BranchKnowledge<J: Justification> {
+    /// ID of the oldest known ancestor if none of them are imported.
+    /// It must be different from the root, as it is always imported.
```

Comment:
`s/as it is/as that is` or something else removing the double "it" referring to different objects.